src/components/Blockquote.tsx

```diff
@@ -24,13 +24,33 @@ export const Blockquote: FunctionComponent<{
     link?: Link
     headline?: string
     largeText?: boolean
-    center?: boolean
     border?: boolean
-}> = ({ quote, author, logo, link, headline, largeText = false, center = true, border = true }) => (
-    <div className={classNames(center && 'text-center')}>
-        <blockquote
-            className={classNames(border && 'border-left border-3 border-vivid-violet', center && 'text-center')}
-        >
+    borderColor?: string
+    inline?: boolean
+}> = ({ quote, author, logo, link, headline, largeText = false, border = true, borderColor, inline = true }) => {
```

Comment:
Note the new props & default values. I think the default values make it simple enough they wont effect blog posts for the average editor. Below will be the default blockquote with no props needed besides text content. Let me know if I should add any more comments in here for documentation!
![image](https://user-images.githubusercontent.com/59381432/181316719-8dcc7833-a355-436e-9f8a-33b54e5ec6e1.png)